doc/generic/pgf/macros/pgfmanual-en-macros.tex

```diff
@@ -18,7 +18,9 @@
 \definecolor{animationgraphicbackground}{rgb}{0.96,0.96,0.8}
 
 \newenvironment{pgfmanualentry}{\list{}{\leftmargin=2em\itemindent-\leftmargin\def\makelabel##1{\hss##1}}}{\endlist}
-\newcommand\pgfmanualentryheadline[1]{\itemsep=0pt\parskip=0pt{\raggedright\item\strut{#1}\par}\topsep=0pt}
+\newcounter{dummy}
+\newcommand\pgfmanualentryheadline[1]{%
+  \itemsep=0pt\parskip=0pt{\raggedright\item\refstepcounter{dummy}\strut{#1}\par}\topsep=0pt}
```

Comment:
Please give the counter a proper name instead of `dummy`.  How about `\newcounter{pgfmanualentry}`?

Reply:
Sounds good.